man/build.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+#
+# A helper script to derive a zeek-client manpage from its argparse state.
+#
+import os.path
+import sys
+
+try:
+    from argparse_manpage.manpage import Manpage
+except ImportError:
+    print("The manpage builder needs the argparse_manpage package.")
+    sys.exit(1)
```

Comment:
I think the default exception text for `ImportError` is already pretty useful, and creating a custom one isn't usually needed, especially for tooling intended for developers.

If you added this to work around `pylint` failing to import this, instead add it to the list of pylint dependencies. Regardless of motivation, you should consider adding the dependency to the pylint pre-commit hook so it can check the code,

```yaml
- repo: https://github.com/PyCQA/pylint
  rev: v3.0.0a7
  hooks:
  - id: pylint
    additional_dependencies:
      - argparse_manpage
      - websocket-client
```